.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,48 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
+        our contributing guidelines for more details on how we assess a brand's
+        popularity.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. For more details on our
+        processes please see our contributing guidelines, which can be found at
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md.
+
+  - type: textarea
+    attributes:
+      label: Additional Comments
+      description: Is there anything else that should be considered for this icon?
```

Definitely think we should keep this field, but I think the description could be improved with some concrete examples of what can be provided here. Any suggestions?

---

I'm thinking we go with "For example: if there are multiple logos that can be used or, if there is a gray area, how this logo complies with the conditions in the contributing guidelines."

---

Sounds good, how about:

```suggestion
      description: |
        Is there anything else we should know about the brand? For example:
        - Are there multiple options for the logo and/or color?
        - Is the icon licensed?
        - If the brand isn't popular enough by the default criteria, how can we asses the popularity instead?
        - If you think the brand might not be accepted for other reasons, why do you think it should be considered?
```